fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/rdf/impl/SkolemNodeRdfContext.java

```diff
@@ -93,6 +94,10 @@ public SkolemNodeRdfContext(final FedoraResource resource,
             return null;
 
         } catch (final RepositoryException e) {
+            if (e instanceof ItemNotFoundException) {
+                // Effectively skip resources that do not exist
```

Comment:
Links to non-existent resources are being skipped because they otherwise result in 404 errors... which is the essence of the ticket associated with this PR.
https://jira.duraspace.org/browse/FCREPO-2323